samples/DurationPickerSample.js

```diff
@@ -0,0 +1,73 @@
+enyo.kind({
+	name: 'moon.sample.DurationPickerSample', components: [
+		{kind: 'FittableColumns',
+		classes: 'moon enyo-unselectable enyo-fit', components: [
+			{kind: 'FittableRows',
+			classes: 'moon enyo-unselectable', fit: true,
+			components: [
+				{kind: 'moon.Divider', content: 'DurationPicker Full Version'},
+				{kind: 'moon.Scroller', fit: true, components: [
+					{classes: 'moon-7h moon-vspacing-s', components: [
+						{kind: 'moon.DurationPicker', name: 'pickerDuration', noneText: 'Pick Duration', content: 'Duration', countdown: true,
+								value: '10:3:30', template: 'h:m:s', onDurationChange: 'durationChanged', onCountdownExpired: 'countdownExpired'},
+						{kind: 'moon.Button', name: 'buttonReset', content: 'Reset Duration', small: true, ontap: 'resetTapped'},
+						{kind: 'moon.Button', name: 'countdownToggle', content: 'Toggle Coutdown', small: true, ontap: 'countdownTapped'}
+					]}
+				]},
+				{kind: 'moon.InputDecorator', components: [
+					{kind: 'moon.Input', placeholder: 'h:m:s', name: 'newTemplate'}
+				]},
+				{kind: 'moon.Button', name: 'setTemplate', content: 'Set Template', ontap: 'setNewTemplate', small: true},
+				{kind: 'moon.InputDecorator', components: [
+					{kind: 'moon.Input', placeholder: '__:__:__', name: 'newValue'}
+				]},
+				{kind: 'moon.Button', name: 'setValue', content: 'Set Value', ontap: 'setNewValue', small: true},
+				{kind: 'moon.Divider', content: 'Result'},
+				{kind: 'moon.BodyText', name: 'result', content: 'Duration Picker - Pick Duration'}
+			]},
+			{kind: 'FittableRows', classes: 'moon enyo-unselectable', components: [
+				{kind: 'moon.Divider', content: 'DurationPicker Disabled'},
+				{kind: 'moon.Scroller', components: [
+					{classes: 'moon-7h moon-vspacing-s', components: [
+						{kind: 'moon.DurationPicker', name: 'pickerDurationDisabled', disabled: true, noneText: 'Pick Duration', content: 'Duration', countdown: true,
+							value: '20:20', template: 'm:s'}
+					]}
+				]},
+				{kind: 'moon.Scroller', fit: true, components: [
+					{kind: 'moon.Divider', content: 'DurationPicker Non Editable'},
+					{classes: 'moon-7h moon-vspacing-s', components: [
+						{kind: 'moon.DurationPicker', name: 'pickerDurationNonEdit', disabledPicker: true, noneText: 'Pick Duration', content: 'Duration', countdown: true,
+							value: '50', template: 's'}
+					]}
+				]}
+			]}
+		]}
+	],
+	create: function(){
+		this.inherited(arguments);
+	},
+	durationChanged: function(sender, ev) {
+		if (this.$.result && ev.value) {
+			this.$.result.set('content', ev.name + ' changed to ' + ev.value);
+		}
+	},
+	countdownExpired: function(sender, ev) {
+		if (this.$.result && ev.name) {
+			this.$.result.set('content', ev.name + ' expired');
+		}
+	},
+	resetTapped: function(sender, ev) {
```

Comment:
I *think* there is a (known?) issue where resetting a picker while it is still open results in the child picker values not being updated until the picker is closed and opened again. You can see this in the `DatePickerSample` as well, which has the same behavior and does not contain an explicit call to close the picker. It's possible that the explicit call to close the picker was an app workaround for this, but I'm unclear on how frequent the case is to reset the picker while it is open. In any case, this might be something to dig into to fix, as a separate issue, and we probably wouldn't want the explicit call to close the picker since it would just be a workaround and not ideal.